### PR TITLE
Create release docs in readme

### PR DESCRIPTION
**Pull request by StevenWessinger**
This adds documentation on releases to the Readme.

### CODE REVIEW

**Comment by StevenWessinger** [comment]
[Create release documentation](https://trello.com/c/zxJclzLr/17-create-release-documentation)

File: README.md [hunk 1]
```diff
@@ -60,6 +60,38 @@ Run the TypeScript linter.
 gulp tslint
 ```
 
+## Releases
+
+Increment all versions within S3Commander to new version (Ex. 0.12.1). Currently
+these are stored under the name s3commander in:
+
+```
+s3commander/packagelock.json
```
Comment (mliudev) [inline README.md]: package-lock.json is an autogenerated file: https://docs.npmjs.com/files/package-lock.json. As such it shouldn't be manually updated.

File: README.md [hunk 2]
```diff
@@ -60,6 +60,38 @@ Run the TypeScript linter.
 gulp tslint
 ```
 
+## Releases
+
+Increment all versions within S3Commander to new version (Ex. 0.12.1). Currently
```
Comment (mliudev) [inline README.md]: This should say something to the effect of increment the s3commander npm package.

**Review by mliudev — CHANGES_REQUESTED** [review]
I'm not comfortable with some of the verbiage you use here and also recommend against changing the package-lock.json file as that's autogenerated.

**Comment by kevinrobbins** [comment]
I think these instructions should also include how to build and deploy to sites. @mliudev Do you agree?

**Pull request merged**